docs/conf.py

```diff
@@ -26,7 +26,7 @@
 # The short X.Y version
 version = ""
 # The full version, including alpha/beta/rc tags
-release = "6.5.1"
+release = "6.5.3"
```

Comment:
I'm not sure what's happening with the keylime bot creating new releases. We should still be on 6.5.1 and I'm going to delete those bad releases.

Reply:
Fixed